pkg/server/streaming.go

```diff
@@ -83,7 +84,10 @@ func (s *streamRuntime) Attach(containerID string, in io.Reader, out, err io.Wri
 }
 
 func (s *streamRuntime) PortForward(podSandboxID string, port int32, stream io.ReadWriteCloser) error {
-	return errors.New("not implemented")
+	if port < 0 || port > math.MaxUint16 {
```

Comment:
The orignal code in kubelet is using `port < 0`. However, you are right, we should check `==0` as well. Will update.